docs/usage.rst

```diff
@@ -221,6 +222,18 @@ It sets the method to load external partial charges.
 
         $ python -m peleffy.main path/to/my_ligand.pdb --charges_from_file path/to/my_ligand.mae
 
+Chain
+-----
+It defines the chain of the molecule to parameterize from the input PDB file.
+
+- Flag: ``--chain``
+- Default: ``None``
```

Comment:
Since the default is None we should explain here the behaviour of the algorithm when this parameter is not initialized. What will happen when no chain is supplied? Then do we need to supply a PDB with a single chain?